src/kitconcept/volto/content_creator/icons.text.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@sneridagh I tried to create an image content object with an SVG image and that failed as well. Just FYI.